src/components/include/utils/threads/thread_delegate.h

```diff
@@ -53,6 +61,10 @@ class ThreadDelegate {
    */
   virtual void threadMain() = 0;
 
+#ifdef QT_PORT
+  Q_SIGNAL void exit_thread();
```

Comment:
As far as I understand, this signal is our custom signal. Why you don't use our code style for the name? Or I miss something?